packages/contracts-bedrock/src/L1/USDXBridge.sol

```diff
@@ -36,6 +36,12 @@ contract USDXBridge is OwnableUpgradeable, ReentrancyGuard, ISemver {
     /// @notice The total amount of USDX bridged via this contract.
     uint256 public totalBridged;
 
+    /// @notice An event emitted when a bridge deposit is made by a user.
+    event BridgeDeposit(address indexed _stablecoin, uint256 indexed _amount, address indexed _to);
```

Comment:
Does it make sense to index the `_amount` field?

Reply:
Will remove the `indexed` modifier